cli/central-client/src/main/java/org/ballerinalang/central/client/CentralAPIClient.java

```diff
@@ -1336,13 +1418,13 @@ public JsonObject getConnector(String id, String supportedPlatform, String balle
         try {
             Request searchReq = getNewRequest(supportedPlatform, ballerinaVersion)
                     .get()
-                    .url(this.baseUrl + "/" + CONNECTORS + "/" + id)
+                    .url(this.baseUrl + SEPARATOR + CONNECTORS + SEPARATOR + id)
```

Comment:
`CONNECTOR_PATH_PREFIX` to hold the value of `SEPARATOR + CONNECTORS + SEPARATOR`?

Reply:
Fixed with fc2c8abcedf